opencti-platform/opencti-front/src/utils/Localization.js

```diff
@@ -8909,6 +8980,14 @@ const i18n = {
       create: 'Creation events',
       update: 'Modification events',
       delete: 'Deletion events',
+      Processing: 'Processing',
+      Automation: 'Automation',
+      'My CSV file contains headers': 'My CSV file contains headers',
+      'CSV separator': 'CSV separator',
+      'Objects found': 'Objects found',
+      'Please contact an administrator': 'Please contact an administrator',
+      'Create a CSV Mapper configuration': 'Create a CSV Mapper configuration',
+      'There are not any configurations set yet': 'There are not any configurations set yet',
```

Comment:
there is no need to translate these messages in english here, we usually use the key as the english message, unless the key is different from the message.

Reply:
I removed these lines